File: user-rest-service-2.0.0-client-java-codegen-develop-2.0/src/main/java/io/swagger/client/model/Location.java
package io.swagger.client.model;



import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;



@ApiModel(description = "")
public class Location  {
  
  private String city = null;
  private String country = null;
  private Double latitude = null;
  private Double longitude = null;
  private String street = null;
  private String zip = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("city")
  public String getCity() {
    return city;
  }
  public void setCity(String city) {
    this.city = city;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("country")
  public String getCountry() {
    return country;
  }
  public void setCountry(String country) {
    this.country = country;
  }

  
  /**
   * demo for java.math.BigDecimal
   **/
  @ApiModelProperty(value = "demo for java.math.BigDecimal")
  @JsonProperty("latitude")
  public Double getLatitude() {
    return latitude;
  }
  public void setLatitude(Double latitude) {
    this.latitude = latitude;
  }

  
  /**
   * demo for java.lang.Double
   **/
  @ApiModelProperty(value = "demo for java.lang.Double")
  @JsonProperty("longitude")
  public Double getLongitude() {
    return longitude;
  }
  public void setLongitude(Double longitude) {
    this.longitude = longitude;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("street")
  public String getStreet() {
    return street;
  }
  public void setStreet(String street) {
    this.street = street;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("zip")
  public String getZip() {
    return zip;
  }
  public void setZip(String zip) {
    this.zip = zip;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Location {\n");
    
    sb.append("  city: ").append(city).append("\n");
    sb.append("  country: ").append(country).append("\n");
    sb.append("  latitude: ").append(latitude).append("\n");
    sb.append("  longitude: ").append(longitude).append("\n");
    sb.append("  street: ").append(street).append("\n");
    sb.append("  zip: ").append(zip).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

File: user-rest-service-1.0.2/generated-code/csharp/src/main/csharp/io/swagger/Model/Location.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;



namespace io.swagger.Model {
  public class Location {
    

    
    public string Street { get; set; }

    

    
    public string Zip { get; set; }

    

    
    public string City { get; set; }

    

    
    public string Country { get; set; }

    

    /* demo for java.math.BigDecimal */
    
    public double? Latitude { get; set; }

    

    /* demo for java.lang.Double */
    
    public double? Longitude { get; set; }

    

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Location {\n");
      
      sb.Append("  Street: ").Append(Street).Append("\n");
      
      sb.Append("  Zip: ").Append(Zip).Append("\n");
      
      sb.Append("  City: ").Append(City).Append("\n");
      
      sb.Append("  Country: ").Append(Country).Append("\n");
      
      sb.Append("  Latitude: ").Append(Latitude).Append("\n");
      
      sb.Append("  Longitude: ").Append(Longitude).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}